fmriprep/workflows/base.py

```diff
@@ -429,7 +429,25 @@ def init_single_subject_wf(subject_id):
         if node.split(".")[-1].startswith("ds_"):
             fmap_wf.get_node(node).interface.out_path_base = ""
 
-    # Step 3: Manually connect PEPOLAR
+    # Step 3: Manually connect PEPOLAR and ANAT workflows
+
+    # Select "MNI152NLin2009cAsym" from standard references.
+    # This node may be used by multiple ANAT estimators, so define outside loop.
+    from niworkflows.interfaces.utility import KeySelect
+    fmap_select_std = pe.Node(
+        KeySelect(fields=["std2anat_xfm"], key="MNI152NLin2009cAsym"),
+        name="fmap_select_std",
+        run_without_submitting=True,
+    )
```

Comment:
can this be inserted into the `if` statement? or does something else depend on this being defined?

Reply:
I thought about this, but then considered that somebody working further down the code might see "this variable is assumed in this loop" and miss that it's only in one condition branch within the loop. Maybe a `NameError` is at least an earlier failure than no input, though...

I can go either way, if you have a strong preference.

Reply:
I'm fine with that train of thought